frontend/exporter/src/types/copied.rs

```diff
@@ -1791,6 +1581,7 @@ pub struct VariantInformations {
 
     pub typ: DefId,
     pub variant: DefId,
+    pub variant_index: VariantIdx,
```

Comment:
I need this to be an option : `None` if the ADT is a structure

Reply:
I removed that option because of the field `typ_is_struct` below: isn't that what you need?

Reply:
Indeed, I hadn't noticed. That works for me.